apm-sniffer/bytebuddy-patch/src/main/java/net/bytebuddy/agent/builder/SWDescriptionStrategy.java

```diff
@@ -354,6 +366,32 @@ public int getModifiers() {
         }
     }
 
+    static class ForLoadedSuperClassWrapper extends TypeDescription.Generic.LazyProjection.ForLoadedSuperClass {
+        private final TypeDescription delegation;
+        private final TypeDescription.Generic superGeneric;
+
+        protected ForLoadedSuperClassWrapper(TypeDescription delegation, TypeDescription.Generic superGeneric) {
+            super(null); // HACK: a trick here since type is not used anywhere in the wrapper
```

Comment:
Has the wrapper overrided all methods of ForLoadedSuperClass? I want to guarantee that the `null` would not be used unexpectedly.

If yes, could you add a UT to check that? This could prevent upgrade issue if byte-buddy core adds a method in the future.